__tests__/Port.test.js

```diff
@@ -0,0 +1,49 @@
+const Itinerary = require('../src/Itinerary.js');
+const Port = require('../src/Port.js');
+const Ship = require('../src/Ship.js');
+
+describe('Port', () => {
+    describe('takes a port and ship', () => {
+        let dublin; 
+        let liverpool; 
+        let itinerary; 
+        let titanic; 
+        
+        beforeEach(() => {
+            liverpool = new Port('Liverpool');
+            dublin = new Port('Dublin');
+            itinerary = new Itinerary([liverpool, dublin]);
+            titanic = new Ship(itinerary);
```

Comment:
This beforeEach block should be in your first describe, so it's more available widely. This way you could delete some of the lines you repeat further below.